cabal.project

```diff
@@ -37,6 +37,19 @@ test-show-details: direct
 -- Always write GHC env files, because they are needed for ghci.
 write-ghc-environment-files: always
 
+-- https://github.com/obsidiansystems/dependent-sum-template/issues/5
+if impl(ghc >= 9.2)
+  constraints :
+      dependent-sum-template < 0.1.2
```

Comment:
Why is this necessary in the build? Please leave a comment otherwise we'll never know when we can remove it.

I know we had a problem here because `plutus-core` pulled in the newer `dependent-sum-template` but I think we can fix that much more nicely by revising `plutus-core` in CHaP (which I've done) rather than propagating such conditionals everywhere.

Reply:
Its a dependency from (I think) plutus-core.

Reply:
I was actually able to remove that conditional.